tools/downloader/converter.py

```diff
@@ -56,19 +56,28 @@ def prefixed_subprocess(prefix, args):
             sys.stdout.write(prefix + ': ' + line)
         return p.wait() == 0
 
+
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
-    prefixed_printf(stdout_prefix, '========= {}Converting {} to ONNX',
-        '(DRY RUN) ' if args.dry_run else '', model.name)
-
-    pytorch_to_onnx_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
-                                                            dl_dir=args.download_dir / model.subdirectory)
-                            for arg in model.pytorch_to_onnx_args]
-    cmd = [str(args.python), str(pytorch_converter), *pytorch_to_onnx_args]
+    converters = {
+        'pytorch': Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
```

Comment:
Given that we already know from #392 that we'll need multiple arguments for conversion from Caffe2, I think you should preemptively support that here (by making the values in this dictionary lists of strings).

Also, please add a trailing comma.

Reply:
That is not true - all conversions will be executed by running actual script, e.g `script.py <params>`. This will standardize conversion parameters list.

Reply:
> That is not true - all conversions will be executed by running actual script, e.g script.py <params>.

But in the case of Caffe2, the "script" is actually `-c <python command>`. That's two arguments.

Reply:
Since we are unifying conversion, I think some conversion parameters should be the same e.g. `input-names`, `input-shape`, while others are framework-specific. This means that we need to have some `script.py` for Caffe2 to parse arguments, to pass them to conversion module.

Reply:
OK, I'm going to drop this, because I don't quite understand your plan and we can always change it later if need be.

I am going to insist on the trailing comma, though. :-)